src/hooks/useActiveOrgV2.tsx

```diff
@@ -0,0 +1,59 @@
+import { getActiveOrgId, setActiveOrgId as setActiveOrgIdCookie } from "@propelauth/javascript"
+import { useContext, useEffect, useState } from "react"
+import { AuthContext } from "../AuthContext"
+
+const ACTIVE_ORG_ID_LOCAL_STORAGE_KEY = "__ACTIVE_ORG_ID"
+
+interface ActiveOrg {
+    activeOrgId?: string
+    setActiveOrgId: (orgId: string) => void
+}
+
+export function useActiveOrgV2(): ActiveOrg | undefined {
```

Comment:
sorry for missing this in the last review - we need to distinguish the loading state from the unset state. If I'm using the hook and get an undefined back, I'm not sure if the data is still being loaded or if an active org was never set.

Reply:
Added a loading boolean to the `ActiveOrg` type